coalib/bearlib/aspects/base.py

```diff
@@ -37,7 +37,9 @@ def __init__(self, language, **taste_values):
                 setattr(self, name, taste_values.get(name, taste.default))
 
     def __eq__(self, other):
-        return type(self) is type(other) and self.tastes == other.tastes
+        return (type(self) is type(other) and
+                self.tastes == other.tastes and
+                self.language == self.language)
```

Comment:
as @rhemon stated, change to ```self.language == other.language```

Reply:
Nah the test failed because currently comparing language still buggy. This still blocked by https://github.com/coala/coala/pull/4109

For example
```py
>>> Language['py 3.3'] == Language['py 3.3']
  ...
  raise InvalidVersion("Invalid version: '{0}'".format(version))
  packaging.version.InvalidVersion: Invalid version: 'Python 3.3'

>>> Language['py'] == Language['py']
  ...
  raise InvalidVersion("Invalid version: '{0}'".format(version))
  packaging.version.InvalidVersion: Invalid version: 'Python 2.7, 3.3, 3.4, 3.5, 3.6'
```